app_feup/lib/controller/local_storage/app_locations_database.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:sqflite/sqflite.dart';
+import 'package:uni/model/entities/location.dart';
+import 'package:uni/model/entities/location_group.dart';
+
+
+
+import 'app_database.dart';
+
+class LocationDatabase extends AppDatabase {
```

Comment:
I think it makes no sense to write to the database, too, since we are reading from a local asset. I think we should just drop this by now.